portfolio/src/main/webapp/index.html

```diff
@@ -93,43 +83,87 @@ <h6 class="card-subtitle text-muted">Technologies Used: Django (Python), HTML/CS
+
                 <small class="text-muted">March 2020</small>
               </div>
             </div>
           </div>
         </div>
         <div class="col-md-4">
           <div class="card mb-4 shadow-sm">
-             <img class="ctr" style="text-align:center;height:225x; width:325px;" src="img/image.png"/>
+             <img class="zoom ctr" style="text-align:center;height:225x; width:325px;" 
+             src="img/image.png"/>
             <div class="card-body">
-              <p class="card-text">Cork. Project for HackedBeta 2019 Hackathon. Made a web app to compile campus posters into a single website where students can filter and sign up for events.</p>
+              <p class="card-text">Cork. Project for HackedBeta 2019 Hackathon. Made a web app to compile campus posters
+                   into a single website where students can filter and sign up for events.</p>
               <h6 class="card-subtitle text-muted">Technologies Used: Django (Python), HTML/CSS, JS</h6>
               <div class="d-flex justify-content-between align-items-center">
-                <a href="https://github.com/katielin42/Django-Web-App-CorkBoard" <button type="button" class="btn btn-sm btn-outline-secondary">Link</button></a>
+                   <a href="https://github.com/katielin42/Django-Web-App-CorkBoard" target="_blank" class="btn btn-sm btn-outline-secondary">Link</a>
                 <small class="text-muted">Nov 2019</small>
               </div>
             </div>
           </div>
         </div>
     
     </div>
-    </body>
-  
-  <script src="https://code.jquery.com/jquery-3.1.1.min.js"></script>
+
+    <br/>
+    <div id="blog">
+    
+        <div id="content">
+
+             <h2 class="center">Blog</h2>
+       <p class="center"><i>What I am currently working on...</i></p>
+       </div>
+            <br/>
+    
+        <div class="blogpost" id="blog0">
+        <h3>Coursera Machine Learning Course</h3>
+        <p class="blogtext">I am currently on Week 6 of Andrew Ng's famous introductory course on machine learning. This week is about 
+            getting proper diagnostics and how to best change parameters to get the best possible fit in your models.  </p>
+        <img src="img/bot.png" class="blogimg"/>
+        </div>
+        <a href="blogs/blog0.html"><button type="button" class="btn btn-sm btn-outline-secondary" style=
+         "margin-left:14px;">Read More</button></a>
+        <br/>
+        <br/>
+
+       <div class="blogpost" id="blog1">
+        <h3>My Portfolio Site </h3>
+        <p class="blogtext">I am currently working on my portfolio page, the page you're currently on right now!</p>
+        <img src="img/site.png" class="blogimg" style="height: 100px;width:150px;"/>
+        </div>
+         <a href="blogs/blog1.html"><button type="button" class="btn btn-sm btn-outline-secondary" style=
```

Comment:
Looks like there's still a few `<a><button>`s on some pages, can you clean them up?

Reply:
Yes, sorry. I noticed another <i> error as well. Will clean up.